cli/cmd/replicaset.go

```diff
@@ -152,6 +152,46 @@ func newExpelCmd() *cobra.Command {
 	return cmd
 }
 
+// newBootstrapVShardCmd creates a "vshard bootstrap" command.
+func newBootstrapVShardCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "bootstrap [--cartridge|--config] [--timeout secs] [flags] " +
```

Comment:
```suggestion
		Use: "bootstrap [--cartridge|--config|--custom] [--timeout secs] [flags] " +
```
`addOrchestratorFlags(cmd)` will add the flag.

Reply:
Added to the descrpition.